### PR TITLE
017 post users user

### PR DESCRIPTION
**Pull request by Zakariyya1**

### CODE REVIEW

File: __test__/app.test.js [hunk 1]
```diff
@@ -293,6 +294,59 @@ describe('/api', () => {
...
+              role: 'Staff'
+            })
+            .then(({ body: { user } }) => {
+              expect(user).toEqual(
+                expect.objectContaining({
+                  user_id: expect.any(Number),
+                  email: expect.any(String),
+                  first_name: expect.any(String),
+                  last_name: expect.any(String),
+                  role: expect.stringMatching(/Staff|Admin/)
```
Comment (joemould) [inline __test__/app.test.js:315]: Nice!

File: controllers/users.controllers.js [hunk 2]
```diff
@@ -1,5 +1,10 @@
...
+const {
+  fetchAllUsers,
+  removeUserById,
+  addUser
+} = require('../models/users.models');
 
+// GET
 const getAllUsers = (req, res, next) => {
   fetchAllUsers()
     .then((users) => {
```
Comment (joemould) [inline controllers/users.controllers.js:10]: need to at some point decide on convention for 200 status codes, whether to implicitly use res.status(200) or leave it out completely

Comment (Zakariyya1) [reply controllers/users.controllers.js:10]: I've made another commit so that all controllers send status code explicitly

**Review by lchapmb — APPROVED** [review]
Looks good

**Review by joemould — APPROVED** [review]

**Pull request merged**